modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/AssignmentsTracker.java

```diff
@@ -307,7 +307,7 @@ private CompletableFuture<TokenizedAssignments> nonEmptyAssignmentFuture(Replica
      *
      * @return Map replication group id to its stable assignments.
      */
-    Map<ReplicationGroupId, TokenizedAssignments> stableAssignments() {
+    public Map<ReplicationGroupId, TokenizedAssignments> stableAssignments() {
```

Comment:
DO we want to make pending assignments getter as public by the way?

Reply:
This getter is required for this patch, but a pending assignment is not.